### PR TITLE
CLI tests

### PR DESCRIPTION
**Pull request by ssk2**
This PR includes a few things:
* Tests (and related fixtures) for the new describe and update commands
* Moved the Cosmos tests to use testify/suite.
* Fixed the plan tests (that were in the wrong place).

### CODE REVIEW

File: cli/commands/update_test.go [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+	// assert CLI output is what we expect
+	expectedOutput := suite.loadFile("testdata/output/update.txt")
+	assert.Equal(suite.T(), string(expectedOutput), suite.capturedOutput.String())
+}
+
+func (suite *UpdateTestSuite) TestUpdateWithEmptyFile() {
+	suite.responseBody = suite.loadFile("testdata/responses/cosmos/1.10/enterprise/update.json")
+	doUpdate("testdata/input/empty.json", "")
+	expectedOutput := suite.loadFile("testdata/output/empty.txt")
```
Comment (nickbp) [inline cli/commands/update_test.go]: For these cases where it's just a one-liner error, could they just be inlined?

e.g. here it's `Failed to load specified options file testdata/input/empty.json: unexpected end of JSON input`

Comment (ssk2) [inline cli/commands/update_test.go]: Sure.

**Comment by benclarkwood** [comment]
Out of curiosity, what is the output of `go test -coverprofile=c.out`? (The percentage)

**Comment by ssk2** [comment]
@benclarkwood:
```➜  commands git:(ssk2/u2-update) ✗ go test -coverprofile=c.out
PASS
coverage: 23.7% of statements
ok  	github.com/mesosphere/dcos-commons/cli/commands	3.478s```

**Comment by ssk2** [comment]
@nickbp: updated.

**Pull request merged**